### ISSUE TITLE
[4.0] com_workflow: Undefined property

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#21551]
### Steps to reproduce the issue
Create new Article, got after Save:

![screen shot 2018-08-11 at 11 13 08](https://user-images.githubusercontent.com/8235763/43990155-ec34607c-9d57-11e8-86e2-47cf1e1fd411.jpg)

### System information
- Nightly Build 4.0.0-alpha5-dev
- Multilingual Sample Data (French, German DE, Persian)
- Template: Cassiopeia
- macOS Sierra, 10.13.6
- Firefox 61 (64-bit)
#### CloudAccess.net
- PHP 7.1.15
- MySQLi 5.7.18-cll-lve

**Comment by ghost** [comment]
There's a second Notice below Article:

![screen shot 2018-08-14 at 13 49 42](https://user-images.githubusercontent.com/8235763/44090030-1501369c-9fc9-11e8-9d03-fe84c876b171.jpg)

**Comment by infograf768** [comment]
Please modify in the file the 2 occurences of `$this->item->condition` to `$this->item->state_condition`

this should solve it.

**Comment by infograf768** [comment]
Code is in ArticlesModel
```
		// Join over the states.
		$query->select('ws.title AS state_title, ws.condition AS state_condition')
			->join('LEFT', '#__workflow_stages AS ws ON ws.id = wa.stage_id');
```

**Comment by ghost** [comment]
Which File(s) i have to modify to look if this solve the Issue?

**Comment by alikon** [comment]
@franz-wohlkoenig test #21621 please

**Comment by ghost** [comment]
closed as having Pull Request #21621
